keyboards/keyspensory/kp60/readme.md

```diff
@@ -0,0 +1,18 @@
+# Keyspensory KP60
+
+Keyspensory KP60 PCB
+
+![Keyspensory KP60 PCB](https://i.imgur.com/3Ob0bp9.png)
```

Comment:
```suggestion
![Keyspensory KP60 PCB](https://i.imgur.com/3Ob0bp9h.png)
```
As per the [`PR Checklist`](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs):
> * if imgur is used, images should be resized appropriately: append “h” to the image url i.e. https://i.imgur.com/vqgE7Ok.jpg becomes https://i.imgur.com/vqgE7Okh.jpg